### DISCUSSION TITLE
Domains / Sub DAOs

### DISCUSSION THREAD

**Discussion by ricardroyo** [iota-community/Community-Governance#23] [General]
Dom suggested the possibility to define certain domain specific DAOs to split up the governance. The idea behind this is that you could vote directly or delegate the vote to other trusted community members on those specific domains. Ideally leading to a situation where you would have domain "experts" with a lot of power in their respective areas.

To keep this discussion on topic, I would keep this thread ONLY to define what could these Domains be. I would keep the discussion of whether this is a good or bad idea or how it should be implemented outside this thread.

**Comment by ricardroyo** [comment]
I'm gonna start by sharing a summary of the type of proposals that Dash has funded historically over the years. We can use this as a draft estimate of the number and types of proposals that the treasury will receive:
![1_-qt-HbEZWDLNeP_QA1DD0A](https://user-images.githubusercontent.com/468015/117446520-79b90800-af3c-11eb-8591-d936149cfd5d.png)

**Reply by ricardroyo** [reply]
I'm gonna say, by the way, that we should not overcomplicate things at the beginning and start with a simple division to start

**Reply by sascha1337** [reply]
DAO of DAOs is a great idea. let me demonstrate you guys some examples. 
but agree @ricardroyo - for the kickstart, we should NOT overcomplicate things - thats very **true** and important. 

I have been looking at other ecosystems, that have running DAOs with onchain treasuries and voting mechanisms last months, with a dream to build this on ISCP. Now _**dom**_ did the unthinkable, and gave us a chance - 

i can only kindly ask for combined workforces to motivate each other to get progress done. **Once-a-lifetime-chance** guys.

I do imagine some kind of self-forming "**Community Workgroups ( Z-Teams? 👍🏻 ) **" with specialists in each fields,
as microDAOs, to succeed faster.

Here some image of a DAO of DAOs / SubDAOs Idea by some smart guy:

![image](https://user-images.githubusercontent.com/12372821/117501988-70b64e00-afb1-11eb-8504-26b8b9a5c7e9.png)
  
Obviously some _Layer1/2/3 crosschain magic_ there, but i like the involvement of those NFTs, and letting ecosystems free flow, very interesting, but futuristic and not for v0 MVP - **for sure.**
